<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editcardialog</class>
 <widget class="QDialog" name="editcardialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>431</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit car</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>New color</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="editCarDialogNewBrand"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="editCarDialogSelectCarID"/>
    </item>
    <item row="8" column="0">
     <widget class="QLineEdit" name="editCarDialogNewColor"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>New brand</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Car ID</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>New transmission</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>New registration number</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="editCarDialogNewRegistrationNumber"/>
    </item>
    <item row="11" column="0">
     <widget class="QComboBox" name="editCarNewTransmission">
      <item>
       <property name="text">
        <string>Automatic</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Manual</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>431</width>
     <height>52</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="editCarDialogSaveEditButton">
      <property name="styleSheet">
       <string notr="true">background: rgb(78, 154, 6)</string>
      </property>
      <property name="text">
       <string>Save Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editCarDialogCancelEditButton">
      <property name="styleSheet">
       <string notr="true">background: rgb(164, 0, 0)</string>
      </property>
      <property name="text">
       <string>Cancel Edit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="editCarDialogConfirmationMessage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
